collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_172.xml

```diff
@@ -41,78 +40,217 @@
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 2v–170r)</locus>
+                        <note> Homilies 1–26</note>
+                        <rubric>Incipit expositio Origenis in librum Numeri </rubric>
+                        <incipit>Divinis numeris non omnes digni sunt</incipit>
+                        <explicit>Ad cuius nos hereditatem perducere dignetur dux et dominus noster Ihesus Christus cui est gloria in secula seculorum amen</explicit>
+                        <note>The first 12 homilies with added running headers ‘om prima’, ‘o ii’, ‘o iii’ etc., ‘o. xxiiii’.</note>
+                        <note>Homily 8 ends on a half-sheet, blank on the verso; Homily 9 begins on a new leaf (fol. 38r).</note>
+                        <note>Baehrens, op. cit. 3–255; Stegmüller, ibid.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 170v)</locus>
+                        <title type="desc" key="work_14541"> (Added) Noted sequence for St. Nicolas of Myrna</title>
+                        <incipit>Laus et gloria deo sit in secula. Qui nobis Nicolaum tradidit sanctum praesulem</incipit>
+                        <explicit>Antistes Nicolaus electus</explicit>
+                        <note>With staveless neumes.</note>
+                        <bibl>pr. <title>Analecta hymnica</title>, liii (1911), pp. 318–19 no. 197, citing 7 MSS. of which 5 are collated, including the present one (siglum ‘B’).</bibl>
+                        <note>Added by a different scribe to the blank last page of a larger-than-normal quire; Homily 27 starts on a new quire.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 171r–188r)</locus>
+                        <note> Homilies 27–28</note>
+                        <rubric>Que sit terrę sanctę descriptio terminorumque eius &amp; finium quod describit dominus</rubric>
+                        <incipit>Cum conderet deus mundum innumeras creavit ciborum differentias </incipit>
+                        <explicit>etiam capilli capitis numerati sunt. Per eum cui est gloria in secula seculorum Amen.</explicit>
+                        <note>Homily 27 with divisions numbered ii–xxxiiii; Homily 28 runs on from it, with only a small initial, as section ‘xxxv’.</note>
+                        <bibl>ed. Baehrens, op. cit., pp. 255–85.</bibl>
+                        <note>These homilies are treated as separate on the label on the front cover; in the edition they are titled ‘De manionibus filiorum Israel’ and ‘Quae sit terrae sanctae descriptio terminorumque eius et finium’, yet in the manuscript the title of the second is given to the first, and the second has no title.</note>
+                        <note>The lower half of fol. 188 excised; fol. 188v blank.</note>
```

Comment:
Should we have a separate `<msItem>` to contain the last `<note>` on the blank page?